contrib/qubes/src/qubes_fwupdmgr.py

```
@@ -627,6 +796,106 @@ def validate_dom0_dirs(self):
+        return updates_list
+
+    def _release_eligible(self, rel_ver_str, cur_ver_str, allow_older, allow_reinstall):
+        """Return True if a release is eligible to install over current using
+        packaging.version human-readable Version strings.
+        """
+        if not rel_ver_str:
+            return False
+        try:
+            rv = pversion.parse(rel_ver_str)
+            cv = pversion.parse(cur_ver_str or "0")
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)